lightwood/analysis/nc/nc.py

```diff
@@ -327,23 +327,23 @@ def score(self, x, y=None):
             Nonconformity scores of samples.
         """ # noqa
         prediction = self.model.predict(x)
-        n_test = x.shape[0]
 
-        norm = np.ones(n_test)
+        err = self.err_func.apply(prediction, y)
```

Comment:
err is not normed here and was before in the return. Is this intended?

Reply:
It is normed bellow ` err = err / norm` we just skip norming when `norm = np.ones(n_test)` aka unit vector aka equivalent to not norming at all... that bit made no sense tbh